lib/pacemaker/pcmk_simulate.c

```diff
@@ -862,6 +866,11 @@ pcmk__simulate(pcmk_scheduler_t *scheduler, pcmk__output_t *out,
 
         cleanup_calculations(scheduler);
         reset(scheduler, input, out, use_date, flags);
+        /* pcmk_unpack_scheduler_input only returns error on scheduler being
+         * NULL or the feature set being unsupported.  Neither of those
+         * conditions could have changed since the first call, so there's no
+         * need to check the return value again.
+         */
```

Comment:
probably still worth checking for bug safety, would let you know quickly something's wrong